README.md

```diff
@@ -4,21 +4,21 @@
 
 Initial scaffolding thanks to: https://github.com/wespr/truffle-vue
 
-1. Install [Truffle](http://truffleframework.com) and an Ethereum client - like [EthereumJS TestRPC](https://github.com/ethereumjs/testrpc). Note I couldn't get this to work on our VMs so I installed them directly on the Mac.
+1. Install [Truffle](http://truffleframework.com) and an Ethereum client - like [Ganache-cli](https://github.com/trufflesuite/ganache-cli).
 	```
 	[sudo] npm install -g truffle // Version 3.0.5+ required.
-	[sudo] npm install -g ethereumjs-testrpc
+	[sudo] npm install -g ganache-cli
```

Comment:
@jsaur what do you think about us defaulting to ganache? seems useful. @johnbayne You've found ganache really useful for getting up to speed, right?

Reply:
Sounds good, it seems like there's not much of a difference but that ganache is what truffle is going with now

Reply:
@coderintherye @jsaur Yes, absolutely. TestRPC redirects to ganache-cli these days - I think ganache has officially superseded it.

Last week we used ganache as opposed to ganache-cli, the above recommends the CLI. Same backend, less mouseclicks, more portability.